### ISSUE TITLE
Right click to view container in browser doesn't always work 

### ISSUE THREAD

**Issue by nebuk89** [microsoft/vscode-docker#1677]
In some scenarios where I would expect right click to work for viewing a container such as the image docker/getting-started when this is running I get no right click option.

**Comment by bwateratmsft** [comment]
@nebuk89 Can you share a screenshot? Looking at the code, the Open In Browser option will only show if the container is running--not stopped, paused, etc. Was that container running when you tried right-clicking?

**Comment by karolz-ms** [comment]
@nebuk89  I cannot reproduce this with VS Code 1.42.1 and Docker extension 0.10.0 running on Ubuntu. What version of VS Code/Docker extension are you using?

**Comment by karolz-ms** [comment]
@nebuk89 I also tried on Windows and I am getting the following error when VS Code attempts to start the container: 
```shell
docker: Error response from daemon: Ports are not available: listen tcp 0.0.0.0:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
```

The command VS Code is using to run the container is 
```shell
docker run --rm -d -p 80:80/tcp docker/getting-started:latest
```
this run command is created based on what ports are exposed by the image.

Suggestions:
1. Use higher-range port for serving the content from this container.
1. Put the tutorial on https://docs.docker.com ☺️ 

So far I do not see anything wrong with the `Open in browser...` command that VS Code Docker extension implements...

@BigMorty FYI

**Comment by bwateratmsft** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our issue reporting guidelines.

Happy Coding!
